lib/tasks/move_supplier.rake

```diff
@@ -0,0 +1,30 @@
+# TODO: Remove this once we've added all suppliers to the all moves in all environments
+namespace :move do
+  desc 'Populates a move with a supplier'
+  task supplier: :environment do
+    total = Move.count
+    total_updated = 0
+
+    locations_with_supplier = Location.joins(:suppliers)
+
+    moves_having_supplier = Move.where.not(supplier_id: nil).count
+    moves_missing_supplier = Move.where.not(from_location: locations_with_supplier).where(supplier_id: nil).count
+
+    locations_with_supplier.each do |from_location|
+      supplier = from_location.suppliers.first
+
+      Move.where(from_location: from_location).where(supplier_id: nil).in_batches(of: 200) do |batch|
+        puts "Populating supplier #{supplier.name} for moves from #{from_location.title}"
+
+        total_updated += batch.update_all(supplier_id: supplier.id)
```

Comment:
since you are using `update_all`, that is quite fast, I am wondering if we can remove `in_batches(of:200)`

Reply:
Still potentially 10k moves for a supplier in production. I think it's better to still use batches, personally